cluster-tools/src/main/scala/org/apache/pekko/cluster/client/ClusterClient.scala

```diff
@@ -56,7 +56,7 @@ import pekko.util.ccompat._
 
 @ccompatUsedUntil213
 @deprecated(
-  "Use Akka gRPC instead, see https://pekko.apache.org/docs/akka/2.6/cluster-client.html#migration-to-akka-grpc",
+  "Use Akka gRPC instead, see https://pekko.apache.org/docs/pekko/2.6/cluster-client.html#migration-to-akka-grpc",
```

Comment:
2.6 is wrong - that is an akka version number - no pekko 2.6 exists

could you check the `migration-to-akka-grpc` bit too? remember that this is pekko, not akka

cluster-client.md may not yet have that header renamed but it should be - could you check?

Reply:
Whole migration section can be deleted and reference `pekko-grpc` link